index.js

```diff
@@ -5,38 +5,48 @@ const request = require('request');
 const rp = require('request-promise-native');
 const wallpaper = require('wallpaper');
 
-const change = () => {
-    const url = `https://www.reddit.com/r/${randomElement(config.subreddits)}/${config.sort}.json?t=${config.from}limit=${config.limit}`;
-    const options = {
-        uri: url,
-        json: true,
-        simple: true,
-    };
+const maxRetry = config.maxRetry
```

Comment:
Lets remove the global `maxRetry` variable since we already have access to `config.maxRetry`.